pkg/speaker/utils.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
-func ParseInterface(ifaceName string, v4 bool) (iface *net.Interface, err error) {
+func ParseInterface(ifaceName string) (iface *net.Interface, err error) {
 	strs := strings.SplitN(ifaceName, ":", 2)
 	if len(strs) == 1 {
 		return net.InterfaceByName(ifaceName)
```

Comment:
When using an IPv6 EIP and specifying the NIC with can_reach, the code may encounter parsing errors.